eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -1,9 +1,11 @@
 parameters:
   ServiceDirectory: not-set
-  ArmTemplateParameters: '@{}'
+  ArmTemplateParameters: '$subscriptionConfiguration.ArmTemplateParameters'
```

Comment:
If we do end up having AdditionalParameters and ArmTemplateParameters you can keep the empty list as the default.